packages/cli/src/Server.ts

```diff
@@ -1753,11 +1755,28 @@ class App {
 			const editorUiPath = require.resolve('n8n-editor-ui');
 			const filePath = pathJoin(pathDirname(editorUiPath), 'dist', 'index.html');
 			const n8nPath = config.getEnv('path');
+			const basePathRegEx = /\/%BASE_PATH%\//g;
 
 			let readIndexFile = readFileSync(filePath, 'utf8');
-			readIndexFile = readIndexFile.replace(/\/%BASE_PATH%\//g, n8nPath);
+			readIndexFile = readIndexFile.replace(basePathRegEx, n8nPath);
 			readIndexFile = readIndexFile.replace(/\/favicon.ico/g, `${n8nPath}favicon.ico`);
 
+			const cssPath = pathJoin(pathDirname(editorUiPath), 'dist', '**/*.css');
```

Comment:
This change means that we'll be keeping a copy of all JS and CSS files in the memory, and increasing the overall memory usage of every instance. Also, this makes gzipping these resources more difficult, which is not really an issue as we are currently not compressing any of our assets (from what I can tell).
However. this isn't a blocker. I can take care of optimizing these and setting up compression in another PR.

Reply:
Thanks a lot, Adi. I've crated a task for it: https://linear.app/n8n/issue/N8N-4739